### PR TITLE
DAT-4081 Restore previous Mercury url building after finished AB test

### PR DESCRIPTION
**Pull request by nikodamn**
- URL building for Mercury
- One Recent Activity test

ping @ludwikkazmierczak @tomasznapieralski @drets @justnpT @Archbold

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/RecentWikiActivityTest.java [hunk 1]
```diff
@@ -19,10 +18,7 @@
 
   @Test(groups = "mercury_recentWikiActivity_anonNavigateToRWAfromMenu")
   public void mercury_recentWikiActivity_anonNavigateToRWAfromMenu() {
-    /**
-     * Temporary change until the end of experiment (21.04.2016) described in DAT-4052.
-     */
-    new Navigate(driver).toPage("/wiki/Aga_Wikia");
+    new MainPage().open();
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/RecentWikiActivityTest.java]: please restore it, as it was https://github.com/Wikia/selenium-tests/blob/36c8ed38c3c2347add58c6ac495eb2401343a80e/src/test/java/com/wikia/webdriver/testcases/mercurytests/RecentWikiActivityTest.java#L21

**Comment by tomasznapieralski** [comment]
1 major, rest is ok

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/RecentWikiActivityTest.java (hunk 1 above)
Comment (nikodamn) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/RecentWikiActivityTest.java]: Resolved

**Comment by SebastianMarzjan** [comment]
Is it good to merge? From the scheduling perspective can get merged ASAP

**Comment by ludwikkazmierczak** [comment]
LGTM

**Pull request merged**